pkg/orchestration/wiring/postgres/plugin.go

```diff
@@ -54,10 +54,15 @@ type LocationSpec struct {
 }
 
 type WiringPlugin struct {
+	Environment func(location voyager.Location) string
 }
 
 func New() *WiringPlugin {
-	return &WiringPlugin{}
+	return &WiringPlugin{
+		Environment: func(_ voyager.Location) string {
+			return "microstestenv"
```

Comment:
shouldn't you accept environment as parameter in constructor instead?

Reply:
For all of them yes.